bids/config.py

```diff
@@ -20,23 +19,48 @@
 
 
 def set_option(key, value):
+    """ Set a package-wide option.
+
+    Args:
```

Comment:
Should we move to numpydoc style here?

And elsewhere.

Reply:
Fine with me if you want to do it... TBH I don't care very much if the docstrings are uniform in style, and if anything I prefer Google style because it's more compact (but I think at this point numpydoc is more common in the codebase). Sphinx will parse both formats fine, so it has no impact on the generated docs.

Reply:
I do not... Anyway, any concerns, or good to merge when tests pass?